extensionsGallery.json

```
@@ -4893,6 +4893,71 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "97",
+					"extensionName": "postgresql-migration",
+					"displayName": "Azure PostgreSQL migration",
+					"shortDescription": "Provides a tool to assess and migrate PostgreSQL database(s) to Azure database for PostgreSQL",
```

(s) is difficult to translate so typically we prefer to just use the plural form

```suggestion
					"shortDescription": "Provides a tool to assess and migrate PostgreSQL databases to Azure database for PostgreSQL",
```